components/about-us/carousel/carousel-3d.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     class="carousel-3d-container my-10 mx-auto"
-    :style="{ height: slideHeight + 'px' }"
+    :style="{ height:slideHeight+150 + 'px' }"
```

Comment:
Don't remove blank lines in this file. They will be added again in somebody else's PR creating unnecessary noise.